tests/Tests/Instances.hs

```diff
@@ -42,7 +42,8 @@ prop_monoidLaw3 opt x =
     in isRight e && mappend a mempty == a
 
 -- mappending two Patterns should be equivalent to appending the original
--- strings they came from and compiling that
+-- strings they came from and compiling that, as long as the second doesn't
+-- start with [.]
```

Comment:
...and those four other excluded cases that this comment doesn't mention. Maybe it's more misleading to mention one case out of five than to not mention any? Just musing, no big deal.

Reply:
Good point, I'll remove it.